app/views/admin/enterprise_roles/_enterprise_role.html.haml

```diff
@@ -1,5 +1,5 @@
 %td {{ enterprise_role.user_email }}
-%td= t('admin.enterprise_roles.manages')
+%td= t('.manages')
```

Comment:
@sauloperez yep! It's called 'Lazy Lookup'
http://guides.rubyonrails.org/i18n.html#looking-up-translations